jobQueue/jobWrapper.js

```diff
@@ -215,7 +273,9 @@ function calculateJobPriority(prioritize, deprioritize, req) {
     low: 'low',
   };
 
-  if (featureToggles.isFeatureEnabled('enableBullForBulkDelSubj')) {
+  if (featureToggles.isFeatureEnabled('enableBullForBulkDelSubj') ||
+   featureToggles.isFeatureEnabled('enableBullForBulkPostEvents') ||
+   featureToggles.isFeatureEnabled('enableBullForCreateAuditEvents')) {
```

Comment:
If we use a single env var as I recommended above, that could also include a toggle for whether at least one bull job is enabled, so we wouldn't have to check every single one here.